tests/res_block_test.py

```diff
@@ -0,0 +1,64 @@
+
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import unittest
+
+from lib import res_block
+
+
+
+class BlockTest(tf.test.TestCase):
+  def setUp(self):
+    super(BlockTest, self).setUp()
+
+  def _run_standard_block(self, input_tensor):
+    block = res_block.ResBlock(kernel_size=3,
+                               expansion_factor=6,
+                               activation='relu')
+    block.build(tf.shape(input_tensor))
+    return block(input_tensor)
+
+  def test_basic(self):
+    """Checking if the input and output tensors shapes match."""
+    input_shape = (32, 128, 128, 64)
+    input_val = tf.random.normal([*input_shape])
+    out = self._run_standard_block(input_val)
+    self.assertShapeEqual(input_val.numpy(), out)
+
+  def test_standard_input(self):
```

Comment:
This test is very similar to `test_basic`, the only difference appears to be the exact shape. I think one of them may be redundant unless we do expect different behavior inside.